features/GGG_CFE_Integration_tests_EmploymentTests/GGG_EMP_7-v5.feature

```diff
@@ -0,0 +1,41 @@
+Feature:
+  " 1. Eligible
+    2. Income contribution
+    3. Employed - monthly "
+
+  Scenario: Test that the correct output is produced for the following set of data.
+    Given I am undertaking a certificated assessment
+    And A submission date of "2020-07-27"
+    And I add the following proceeding types in the current assessment:
+      | ccms_code | client_involvement_type |
+      | DA001     | A                       |
+    And I have a dependant aged 3
```

Comment:
Unless my maths is wrong should they be 1 (if we are rounding to a full year)?

Reply:
fixed, although for these tests the specific age doesnt matter as long as they are of an age that childcare is a legitimate expense so anything pre-school age should be fine